reactions.json

```diff
@@ -17,7 +17,7 @@
     "Cl / uv": "2-chloropropane",
     "Br / uv": "2-bromopropane"
   },
-  "chloromethane": {
+  "1-chloromethane": {

```

Comment:
chloromethane should be preferred over 1-chloromethane since it's redundant to write that 1